app/src/main/java/edu/uc/it3048/mypokedex/ui/main/MainFragment.kt

```diff
@@ -10,21 +10,24 @@ import edu.uc.it3048.mypokedex.R
 
 class MainFragment : Fragment() {
 
-    companion object {
-        fun newInstance() = MainFragment()
-    }
-
     private lateinit var viewModel: MainViewModel
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?,
-                              savedInstanceState: Bundle?): View {
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View {
+
         return inflater.inflate(R.layout.main_fragment, container, false)
     }
 
     override fun onActivityCreated(savedInstanceState: Bundle?) {
+
         super.onActivityCreated(savedInstanceState)
-        viewModel = ViewModelProviders.of(this).get(MainViewModel::class.java)
+        viewModel = ViewModelProvider(this).get(MainViewModel::class.java)
+
         // TODO: Use the ViewModel
     }
 
+    companion object {
+
```

Comment:
Moved companion object to the bottom per: https://kotlinlang.org/docs/reference/coding-conventions.html